pkg/test/resourcefixture/testdata/basic/backupdr/v1alpha1/backupdrbackupplanassociation/backupdrbackupplanassociation-minimal/create.yaml

```diff
@@ -20,7 +20,6 @@ spec:
   projectRef:
     external: ${projectId}
   location: us-central1
-  description: "Initial description"
```

Comment:
Is this intended? we don't want to remove field from test coverage

Reply:
Yeah, the nice thing about SSA is it will flag fields that are invalid. This was overlooked.